pkgs/development/python-modules/grep-ast/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+
+  pathspec,
+  tree-sitter-languages,
+}:
+
+buildPythonPackage rec {
+  pname = "grep-ast";
+  version = "0.3.3";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "grep_ast";
+    hash = "sha256-QriIfVcwHcVWNDaPjVSenEnJE9r7TRnJtUw922BPzPQ=";
+  };
+
+  dependencies = [
+    pathspec
+    tree-sitter-languages
+  ];
+
```

Comment:
Could you enable tests with `pythonImportsCheck` and `pytestCheckHook`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

Reply:
Done